enh: add policies and terms (#19)
dandi/handbook: docs/about/policies.md

  as [Neurodata Without Borders](https://nwb.org),[Brain Imaging Data Structure](https://bids.neuroimaging.io/),
  [Neuroimaging Data Model](NIDM) and other [BRAIN Initiative](https://braininitiative.nih.gov/)
  standards. We are working with the community to improve these standards and to
  improve metadata to make the datasets more FAIR.
- **Volume and size limitations:** There is a limit of 5TB per asset and we currently
  accept any size of standardized datasets, as long as you can upload them over
  an HTTP connection. However, if you plan to upload more than 50TB of data please
  get in touch with us.
- **Data quality:** All information is provided “as-is”, and the user shall hold
  DANDI and information providers supplying data to Zenodo free and harmless in
  connection with the use of such information.
- **Metadata types and sources:** All metadata is stored internally in JSON-format
  according to a defined JSON schema. Metadata is exported in several standard
  formats such as JSON-LD and DataCite Metadata Schema.
- **Language:** For textual items, English is preferred but all languages are
  accepted.

> - **Licenses:** Users must specify a license for each dataset.

Reviewer: with the note on metadata below, I would even further extend it to 
```suggestion
- **Licenses:** Users must specify a license for each dataset chosen from the list of the DANDI archive approved licenses. Users allow for the DANDI archive to extract metadata records and make them available under permissive CC0 license.
```

## Access and Reuse

- **Access to data objects:** Files may be deposited under open <!--or embargoed-->
  access. Access to metadata and data files is provided over standard protocols
  such as HTTP.
- **Use and re-use of data objects:** Use and re-use is subject to the license
  under which the data objects were deposited.

> - **Metadata access and reuse:** Metadata is licensed under CC0, except for email
>   addresses. All metadata is exported via HTTP and can be harvested.

Reviewer: AINAL, but a bit tricky here
- factual "data" items (think - subject_id) might not even be copyrightable and thus not regulated by "License" terms. The same for email addresses -- alone, they aren't copyrightable AFAIK, thus we can't state that we have a license X for metadata but not for email addresses.
- collections of (meta)data, such as dataset and even NWB is a different topic
- we extract metadata for assets, which might be under non-CC0 license: we should somehow allow for such "derivative re-licensing".
- I would not mix in access protocols in here...

what about:

```suggestion
- **Metadata access and reuse:** Metadata records, provided by the users or extracted from the assets, are licensed under CC0. All metadata is made publicly available and can be harvested.
```

>   access. Access to metadata and data files is provided over standard protocols

Reviewer 2: > Files may be deposited under open access.

This reads oddly.

<!--
- **Embargo status:** Users may deposit content under an embargo status and
  provide an anticipated end date for the embargo. The repository will restrict
  access to the data until the end of the embargo period; at which time, the
  content will become publically available automatically. The end of the embargo
  period is the earliest of the date provided by submitter, the first publication
  using the data, or the end of funding support for the collection and/or disemmination
  of the dataset.
- **Restricted Access:** Depositors of embargoed datasets have the ability to
  share access with other collaborators. These files will not be made publicly
  available till the end of the embargo period.
-->

## Removal

- **Revocation:** Content not considered to fall under the scope of the repository

>   will be removed and associated DOIs issued by DANDI revoked. Please signal
>   promptly, ideally no later than 24 hours from upload, any suspected policy
>   violation. Alternatively, content found to already have an external DOI will

Reviewer: ```suggestion
  can be removed and associated DOIs issued by DANDI revoked. Please inform DANDI team
  promptly, ideally no later than 24 hours from upload, about any suspected policy
  violation. Alternatively, content found to already have an external DOI will
```

  have the DANDI DOI invalidated and the record updated to indicate the original
  external DOI. User access may be revoked on violation of Terms of Use.
<!--
Enable after publish is in place
- **Withdrawal:** If the uploaded research object must later be withdrawn, the
  reason for the withdrawal will be indicated on a tombstone page, which will
  henceforth be served in its place. Withdrawal is considered an exceptional
  action, which normally should be requested and fully justified by the original
  uploader. In any other circumstance reasonable attempts will be made to contact
  the original uploader to obtain consent. The DOI and the URL of the original
  object are retained.
-->

> - **User data on Dandihub:** At present user data on Dandihub will be removed at
>   periodic intervals and should not be seen as persistent.

Reviewer: ```suggestion
- **User data on Dandihub:** At present, user data on Dandihub is be removed
  periodically and Dandihub storage space should not be considered persistent.
```

## Longevity

- **Versions:** Datasets are versioned when published. Prior to publishing the

>   state of a dataset may continue to evolve and the data or metadata are not
>   versioned. Derivatives of data files may be generated, but original content is

Reviewer: ```suggestion
  state of a dataset may continue to evolve and the data or metadata are neither
  versioned, nor guaranteed to persist. Derivatives of data files may be generated, but original content is
```

  never modified.
- **Replicas:** All data files are stored on an AWS public bucket, with replicas
  housed at Dartmouth College.  Data files are kept in multiple replicas at the

>   moment, but this may change over time.

Reviewer: ```suggestion
  moment, but this may change over time, and no recovery mechanisms for not versioned data
  should be assumed to be in place.
```

> - **Retention period:** Items will be retained for the lifetime of the repository.

Reviewer: ```suggestion
- **Retention period:** Versioned items will be retained for the lifetime of the repository.
```

  This is currently the lifetime of the NIH award, which currently expires in
  April 2024.
- **Functional preservation:** DANDI makes no promises of usability and

>   understandability of deposited objects over time.

Reviewer: ```suggestion
  understandability of deposited objects.
```
not sure what "over time" really means in the context

- **File preservation:** Data files and metadata are backed up nightly and

>   replicated into multiple copies in the online system.

Reviewer: should be somehow merged with Replicas I think. Otherwise it is just not clear what is "online system" and then contingent on "Replicas" which are disclosed to possibly change in the future

>   replicated into multiple copies in the online system.

Author: ```suggestion
  replicated into multiple copies in different storage services.
```

> - **Fixity and authenticity:** All data files are stored along with multiple

Reviewer: note: first time I hear about "Fixity"... might be just me

>   moment, but this may change over time, and no recovery mechanisms for not versioned data

Reviewer 2: ```suggestion
  moment, but this may change over time, and no recovery mechanisms for unversioned data
```

  checksum of the file content. Files are regularly checked against their
  checksums to assure that file content remains constant.

> - **Succession plans:** In case of closure of the repository, best efforts will

Reviewer: ```suggestion
- **Succession plans:** In case of the repository shut down, best efforts will
```

  be made to integrate all content into suitable alternative institutional and/or

>   subject based repositories.

Reviewer: ```suggestion
  other repositories overlapping inthe scope of the DANDI archive.
```
since the notion of "subject" was not articulated yet.

>   checksum of the file content. Files are regularly checked against their

Reviewer 2: ```suggestion
  checksums of the file content. Files are regularly checked against their
```
